sbt-bridge/src/dotty/tools/xsbt/PositionBridge.java

```diff
@@ -116,4 +116,9 @@ public String lineContent() {
       result.append(lineContent.charAt(i) == '\t' ? '\t' : ' ');
     return Optional.of(result.toString());
   }
+
+  @Override
+  public String toString() {
+      return "PositionBridge(" + pos+ ")";
```

Comment:
I'm fine with making the toString whatever it needs to be to match the behavior you had with scala 2, no need for a "PositionBridge()" around

Reply:
I thought to follow the convention of other code around it, e.g. https://github.com/lampepfl/dotty/blob/master/sbt-bridge/src/dotty/tools/xsbt/Problem.java.
But happy to remove it, if preferred?

Reply:
I removed wrapper, so that it matches behavior of scala 2 more closely.
It should look now like:
`/../XY.scala:[3634..3642..3657]`